.github/ISSUE_TEMPLATE/dashboard_issue.yml

```diff
@@ -0,0 +1,98 @@
+          required: false
+        - label: Is this a support query related to your specific use case? Contact us at [support@arcana.network](mailto:support@arcana.network).
+          required: false
+  - type: markdown
+    attributes:
+      value: "## Create GitHub Issue for Arcana Dashboard"
+  - type: markdown
+    attributes:
+      value: |
+
+        Make sure to add all the information needed to understand the bug or your feature request.
+
+        If the info is missing we'll add the 'Needs more information' label and prioritize it only when there is enough information provided.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Describe the scenario and the bug.
+      placeholder: Tell us what you see!
+      value: "Something did not work when you tried..."
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: Tell us, what did you expect to happen?
+      placeholder: Tell us about the expected behavior!
+      value: "This error should show up or this value should be returned, etc."
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: "## Help us Recreate the Bug"
+  - type: textarea
+    id: steps
+    attributes:
+      label: List the Steps
+      description: Describe how to reproduce the bug step by step.
+      placeholder: Details
+      value: |
+         1. Do this...
+         2. Next do this...
+         3. ...
+    validations:
+        required: true
+  - type: markdown
+    attributes:
+      value: |
+        * Provide screenshots where appropriate
+        * Provide a minimal code snippet / [rnplay](https://rnplay.org/) example that reproduces the bug.
+  - type: textarea
+    id: dev-env
+    attributes:
+      label: Development Environment
+      description: Describe your development environment package version and other details.
+      placeholder: placeholder data
+      value: |
+        * What version of Arcana Dashboard you are using?
+        * If you are using Arcaan Auth SDK for your dApp, mention its Version.
+        * If you are using Arcana Storage SDK, mention its Version.
+        * What's the version of React Native / Vue / JavaScript / TypeScript you're using in your dApp?
+        * Does this issue show up on iOS, Android or both mobileplatforms?
+        * Are you using Mac, Linux or Windows desktop platform?
+        * Which browser / version you were using to access the Dashboard? 
+    validations:
+        required: false
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more information about your bug?
+      placeholder: ex. youremail@youremailprovider.com
+    validations:
+      required: false
```

Comment:
IMO, we can skip this. Not everyone will be comfortable providing their contact details on a public forum and  this will take the discussion away from the GH issue. We can always ask for contact details if there's a specific need for that issue.

Reply:
ok